### PR TITLE
[zh] Translate blog post 2021-12-22-kubernetes-in-kubernetes-and-pxe-bootable-server-farm.md into Chinese

### PR DESCRIPTION
**Pull request by taniaduggal**
…etes-and-pxe-bootable-server-farm.md into Chinese

 Translated blog/_posts/2021-12-22-kubernetes-in-kubernetes-and-pxe-bootable-server-farm.md into Chinese #31449

### CODE REVIEW

**Comment by NitishKumar06** [comment]
@taniaduggal Please make sure to assign yourself to this PR as well . It'd be more better.

File: 2021-12-22-Kubernetes-in-kubernetes-and-pxe-bootable-server-farm.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
+---
+佈局：博客
+標題：“Kubernetes-in-Kubernetes 和 WEDOS PXE 可啟動服務器場”
+蛞蝓：kubernetes-in-kubernetes-and-pxe-bootable-server-farm
+日期：2021-12-22
```
Comment (Sea-n) [inline 2021-12-22-Kubernetes-in-kubernetes-and-pxe-bootable-server-farm.md:5]: The metadata key cannot be translated.
```suggestion
佈局：博客
標題：“Kubernetes-in-Kubernetes 和 WEDOS PXE 可啟動服務器場”
蛞蝓：kubernetes-in-kubernetes-and-pxe-bootable-server-farm
日期：2021-12-22
```

**Review by Sea-n — COMMENTED** [review]
Hum... Are you a Chinese speaker? Or at least can read basic Chinese sentence?
If so, please refer to [中文本地化樣式指南](https://kubernetes.io/zh/docs/contribute/localization_zh/).

There are many formatting and wording errors in this PR, I'd suggest that we can review after you read the guideline and fix the basic errors.
/hold

**Comment by reylejano** [comment]
Hi @kubernetes/sig-docs-zh-owners and @kubernetes/sig-docs-zh-reviews , please take a look at this PR

**Review by Adarsh-jaiss — APPROVED** [review]

**Comment by taniaduggal** [comment]
hey @Sea-n . I have resolve the conflicts .Please review it once again .

**Comment by Sea-n** [comment]
Wait... Did you translate `/content/en/blog/_posts/2021-12-22-kubernetes-in-kubernetes-and-pxe-bootable-server-farm.md` to the root directory `/2021-12-22-Kubernetes-in-kubernetes-and-pxe-bootable-server-farm.md`? 🤔

For Chinese blog post, please put it in `/content/zh/blog/_posts/2021-12-22-kubernetes-in-kubernetes-and-pxe-bootable-server-farm.md`.

**Pull request closed without merging**